docs/Troubleshooting.md

```diff
@@ -66,19 +66,20 @@ If you are organizing your epics into a class. (E.g. in order to benefit from An
 
 ```typescript
 class TooFancy {
-  constructor(private somethingInjected:SomethingInjected)
-  checkAutoLogin (action$: Observable<IPayloadAction>) {
+  constructor(private somethingInjected: SomethingInjected) {}
+  checkAutoLogin = (action$: Observable<IPayloadAction>) => {
     console.log(this); // Is Window! when called from redux-observable
   }
 }
 ```
+
 follow the docs and:
 
 ```typescript
 class TooFancy {
-  constructor(private somethingInjected:SomethingInjected)
-  checkAutoLogin =  (action$: Observable<IPayloadAction>) => {
-    console.log(this); // YOu can access somethingInjected
```

Comment:
Hmm I believe there is some confusion. I believe that section is saying you NEED to use arrow functions so that you close over the `this` of the class instance. The example is showing the before/after. It probably can be reworded to make that more clear though. Lmk if I'm misunderstanding.